examples/website/collision-filter/app.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import {createRoot} from 'react-dom/client';
+import {Map} from 'react-map-gl';
+import maplibregl from 'maplibre-gl';
+import DeckGL from '@deck.gl/react';
+import {GeoJsonLayer, TextLayer} from '@deck.gl/layers';
+import points from './data/ne_10_roads_filtered_usa_10km_pts_california.json';
+import roads from './data/ne_10_roads_filtered_usa_california.json';
+import {CollisionFilterExtension} from '@deck.gl/extensions';
+import * as turf from '@turf/turf';
+
+const ALL_ROUTES = 'All routes';
+
+export default function App({
+  mapStyle = 'https://basemaps.cartocdn.com/gl/dark-matter-nolabels-gl-style/style.json',
+  sizeScale = 4,
+  collisionEnabled = true,
+  routeName = 'US-101',
+}) {
+  const roadName = routeName.split('-');
+  const filteredRoads =
+    routeName === ALL_ROUTES
+      ? roads
+      : roads.features.filter(f => f.properties.number === roadName[1]);
+  const filteredLabels =
```

Comment:
- Labels should be generated from the road geometry, not loaded from a GeoJSON
- The angles do not look correct

Reply:
Fixing angle
@cayetanobv `Labels should be generated from the road geometry, not loaded from a GeoJSON`

Reply:
FYI codepen to align label with path: https://codepen.io/Pessimistress/pen/OJgmXba?editors=0010